infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/AddNode.java

```diff
@@ -0,0 +1,79 @@
+package org.corfudb.infrastructure.orchestrator.workflows;
+
+import lombok.Getter;
+import org.corfudb.infrastructure.orchestrator.Action;
+import org.corfudb.infrastructure.orchestrator.Workflow;
+import org.corfudb.runtime.CorfuRuntime;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * A workflow that adds a new node to the cluster.
+ *
+ * Created by Maithem on 10/25/17.
+ */
+
+public class AddNode extends Workflow {
+
+    @Getter
```

Comment:
The workflow doesn't seem to do anything...

Reply:
Correct. Its a stub for now. I just added these stubs to illustrate the use of model.

Reply:
Well, it needs some description if it's going to be part of this PR because I don't understand how it works.

Reply:
I'll add more documentation.